src/python/doc/representations_tflow_itf_ref.rst

```diff
@@ -0,0 +1,127 @@
+:orphan:
+
+.. To get rid of WARNING: document isn't included in any toctree
+
+====================================
+TensorFlow layer for representations
+====================================
+
+.. include:: representations_sum.inc
+
+`PersLay <http://proceedings.mlr.press/v108/carriere20a.html>`_ is a general neural network layer for performing deep
+learning with persistence diagrams, implemented in TensorFlow.
```

Comment:
I would be a bit more descriptive here:
"`PersLay <http://proceedings.mlr.press/v108/carriere20a.html>`_ is a layer for neural network architectures that allows to automatically learn the best representation to use for persistence diagrams in supervised machine learning during training time. Its parameters allow to reproduce most of the known finite-dimensional representations (such as, e.g., landscapes and images), and can be combined to create even new ones, that are best suited for a given supervised machine learning task with persistence diagrams. PersLay is implemented in TensorFlow."

Reply:
I added your proposition on [ee00c3e](https://github.com/GUDHI/gudhi-devel/pull/786/commits/ee00c3e421a5f896218e11cf3f0d47af9b134053)